<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="widgetSerial" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacerSerial3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="labelSerialName">
      <property name="text">
       <string>串口名字：</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacerSerial5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QComboBox" name="comboBoxSerialCom"/>
    </item>
    <item>
     <spacer name="horizontalSpacerSerial1">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QComboBox" name="comboBoxSerialBaundRate"/>
    </item>
    <item>
     <spacer name="horizontalSpacerSerial2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonSerialSwitch">
      <property name="text">
       <string>打开串口</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacerSerial4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonSerialSend">
      <property name="text">
       <string>发送数据</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacerSerial6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>291</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBoxBusInteriorDevice">
      <property name="title">
       <string>车内设备</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="radioButtonFrontScreen">
         <property name="text">
          <string>前内屏</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonBackScreen">
         <property name="text">
          <string>后内屏</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonFrontRoad">
         <property name="text">
          <string>前路牌</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonSideRoad">
         <property name="text">
          <string>侧路牌</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonBackRoad">
         <property name="text">
          <string>后路牌</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonFrontSection">
         <property name="text">
          <string>前站节牌</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonBackSection">
         <property name="text">
          <string>后站节牌</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonStopAnnouncer">
         <property name="text">
          <string>报站器</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxFrontScreen">
      <property name="title">
       <string>前内屏功能</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="radioButtonFrontScreenSelfCheck">
         <property name="text">
          <string>自检功能</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonFrontScreenSetTime">
         <property name="text">
          <string>时间设置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonFrontScreenSetStopName">
         <property name="text">
          <string>站名设置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonFrontScreenSetFirstLast">
         <property name="text">
          <string>首末句设置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonFrontScreenSetPropagandaLanguage">
         <property name="text">
          <string>宣传用语设置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonFrontScreenPlayStopName">
         <property name="text">
          <string>站名播放</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonFrontScreenPlayNumProLanguage">
         <property name="text">
          <string>宣传用语按序号播放</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonFrontScreenPlayItemProLanguage">
         <property name="text">
          <string>宣传用语按内容播放</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxBackScreen">
      <property name="title">
       <string>后内屏功能</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="radioButtonBackScreenSelfCheck">
         <property name="text">
          <string>自检功能</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonBackScreenSetTime">
         <property name="text">
          <string>时间设置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonBackScreenSetStopName">
         <property name="text">
          <string>站名设置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonBackScreenSetFirstLast">
         <property name="text">
          <string>首末句设置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonBackScreenSetPropagandaLanguage">
         <property name="text">
          <string>宣传用语设置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonBackScreenPlayStopName">
         <property name="text">
          <string>站名播放</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonBackScreenPlayNumProLanguage">
         <property name="text">
          <string>宣传用语按序号播放</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonBackScreenPlayItemProLanguage">
         <property name="text">
          <string>宣传用语按内容播放</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textEditTxRxDataDisplay">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>601</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
